src/zenml/integrations/kubernetes/orchestrators/kube_utils.py

```diff
@@ -300,13 +300,15 @@ def create_edit_service_account(
         cluster_role_binding_name: Name of the cluster role binding.
             Defaults to "zenml-edit".
     """
-    crb_manifest = build_cluster_role_binding_manifest_for_service_account(
+    crb_manifest = build_role_binding_manifest_for_service_account(
```

Comment:
Is this now a `rb_manifest` if it's not a cluster role binding anymore? Also the `cluster_role_binding_name` seems wrong?

Reply:
I did another round of renaming. Should be good now.